animations/test/misc/focus_image_test.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:animations/src/misc/focus_image.dart';
+
+Widget createFocusImageScreen() => MaterialApp(
+      home: FocusImageDemo(),
+    );
+
+void main() {
+  group('FocusImage Tests', () {
+    testWidgets('Size of ink well changes on tap', (tester) async {
+      await tester.pumpWidget(createFocusImageScreen());
+
+      // Get the initial inkwell.
+      final initialInkwell = tester.firstWidget(find.byType(InkWell));
```

Comment:
It's probably not an issue, but this test is using two different methods to find the first InkWell widget (`firstWidget` and `.at(0)`).

For consistency and readability, convert the searches on line 19 and 29 to use:

tester.widget(find.byType(InkWell).at(0))

Reply:
sure thing!